packages/shared/components/GradientPicker.svelte

```diff
@@ -1,32 +1,37 @@
 <script type="typescript">
-    import { createEventDispatcher } from 'svelte';
-	import { fade, fly } from 'svelte/transition';
-	import { quintInOut } from 'svelte/easing';
-	
-    const dispatch = createEventDispatcher();
-    
+    import { createEventDispatcher } from 'svelte'
+    import { fade, fly } from 'svelte/transition'
+    import { quintInOut } from 'svelte/easing'
+
+    const dispatch = createEventDispatcher()
+
     function hsl2rgb(h: number, s: number, l: number) {
         let a = s * Math.min(l, 1 - l)
-        let f = (n, k = ( n + h / 30) % 12) => l - a * Math.max(Math.min(k - 3, 9 - k, 1), - 1)
-        return [ f(0), f(8), f(4) ]
+        let f = (n, k = (n + h / 30) % 12) => l - a * Math.max(Math.min(k - 3, 9 - k, 1), -1)
```

Comment:
I have no clue what's happening here, but GH is complaining that it should be a const. Let's make it a const then. Maybe also some proper variable naming is in place here, so the reader at least has a clue what's happening.

Reply:
Thanks you both! Added some comments and the URL for reference. I think it's better to keep the variable names since it's a math formula and the reference it's perfect to add clearness.